### PR TITLE
update token retry and send to sentry

### PR DESCRIPTION
**Pull request by engineersamuel**

### CODE REVIEW

File: src/jwt.ts [hunk 1]
```diff
@@ -486,12 +499,13 @@ function isTokenExpired(): boolean {
...
 /**
- * Refreshes the access token.
+ * Refreshes the access token.  Recursively can be called with an iteration count
+ * where the function will retry x number of times.
  *
  * @memberof module:session
  * @private
  */
-function updateToken(force?: boolean): ISimplePromise {
+function updateToken(force: boolean = false, iteration: number = 0): ISimplePromise {
```
Comment (engineersamuel) [inline src/jwt.ts]: @mwcz Relevant part of token retry.

File: src/jwt.ts [hunk 2]
```diff
@@ -731,7 +768,7 @@ function getUserInfo(): IJwtUser {
     // the properties to return
     const token = getToken();
     return token ? {
-        user_id: token.user_id,
+        id: token.user_id,
```
Comment (mwcz) [inline src/jwt.ts]: Please leave `user_id` intact.  This user info object has been around for years and changing it will break all sorts of things.

Comment (engineersamuel) [reply src/jwt.ts]: I will add it back in, note as well that Sentry does not directly read a field named user_id but id: https://docs.sentry.io/learn/context/#capturing-the-user

Comment (engineersamuel) [reply src/jwt.ts]: Done.

**Pull request merged**